back-end/src/nova/main.go

```diff
@@ -2,21 +2,35 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/jmcrumb/nova/accounts"
+	"github.com/jmcrumb/nova/auth"
+	"github.com/jmcrumb/nova/database"
 	"github.com/jmcrumb/nova/nlp"
 )
 
 func main() {
+	err := database.SetupDB()
+	if err != nil {
+		panic("failed to connect database")
+	}
+
 	router := gin.Default()
 
+	// route authorization API
+	authGroup := router.Group("/auth")
+	auth.Route(authGroup)
+
 	// route NLP api
-	group := router.Group("/nlp")
-	nlp.Route(group)
+	nlpGroup := router.Group("/nlp")
+	nlp.Route(nlpGroup)
 
 	// route account database
+	accountGroup := router.Group("/account")
+	accounts.Route(accountGroup)
 
 	// route plugin store api
 
-	router.SetTrustedProxies([]string{"localhost"})
+	router.SetTrustedProxies([]string{"localhost", "127.0.0.1"})
```

Comment:
I am unsure why 127.0.0.1 is used or would change beyond testing. Just curious though.